src/server-settings.ts

```diff
@@ -8,7 +8,7 @@ export function getSetting(setting: string, deflt?: any): any {
     throw new Error('Settings have not been loaded. Call load() first');
   }
 
-  const value = settingsCache.get(setting);
+  const value = JSON.parse(settingsCache.get(setting));
```

Comment:
JSON.parse should be called when loading the settings from the database. Doing it there means JSON.parse only happens once, not every time getSetting is called.